HISTORY.md

```diff
@@ -0,0 +1,21 @@
+Anserini
+========
+
+## v0.1.0
+
++ Release date: July 4, 2018
+
+Initial release. Fully-functional, repeatable regression tests for the following TREC experiments:
+
++ Experiments on Disks 1 &amp; 2
++ Robust04 experiments on Disks 4 &amp; 5
++ Robust05 experiments on the AQUAINT collection
++ CORE17 experiments on the New York Times collection
++ TREC Microblog 2011-2014 experiments
++ Experiments on Wt10g collection
++ Experiments on Gov2 collection
++ Experiments on ClueWeb09b collection
++ Experiments on ClueWeb12-B13 collection
++ Experiments on ClueWeb12 collection
+
+Working ranking models include QL, BM25, and RM3.
```

Comment:
I think it is also possible to list all the contributors (github usernames) for THIS release.
We can keep the list onward for the contributors just for a release? We can get the creators of PRs from the project page, right?